src/lib/geojson-helpers.js

```diff
@@ -0,0 +1,11 @@
+import union from '@turf/union'
+
+export function mergeFeatures (features) {
+  return features.reduce((acc, feature) => {
+    if (!acc) {
+      return feature
+    }
+
+    return union(acc, feature)
+  }, undefined)
+}
```

Comment:
Merges multiple features into one, as valid GeoJSON.

Reply:
Looks good. 

Would the following also work?:
`features.reduce((acc, feature) => union(acc, feature), features[0])`
But then you would have to check length of features first....

Reply:
Yeah that would make the code a lot clearer I think. Adjusted in 6229f418cd4ca495611293ca3dc9d56b338ca5fa